docs/2.build/1.chain-abstraction/what-is.md

```diff
@@ -1,61 +1,96 @@
 ---
 id: what-is
-title: What is Chain Abstraction? 
+title: What is Chain Abstraction?
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
-import {CodeTabs, Language, Github} from "@site/src/components/codetabs"
+import {CodeTabs, Language, Github} from "@site/src/components/codetabs";
 
 The idea behind `chain abstraction` is quite simple: **blockchain** technology should be **abstracted away** from the user experience. In other words, people should **not realize** when they are **using a blockchain**, nor **which blockchain** they are using.
 
 ![img](/docs/assets/welcome-pages/chain-abstraction-landing.png)
 
-To help on this task, NEAR Protocol provides services that allow to **create** and **recover accounts** using **email addresses**, use the account **without acquiring funds**, and **control accounts** in **other chains**. All in the most **seamless** way possible.
+To help on this task, NEAR Protocol provides services that allow to **create** and **recover accounts** using **email addresses**, use the account **without acquiring funds**, and **control accounts** in **other chains**. The combination of these services allows to create a **seamless** user experience, in which users can use blockchain-based applications without realizing they are using a blockchain.
 
----
+As an example, this would allow users to collect NFTs across different chains, without ever needing to explicitly create an account or acquire crypto. All with just a single login.
 
-## Relayers: Cover gas fees
-Allowing users to start using a dApp without having to acquire funds is a powerful tool to increase user adoption. NEAR Protocol provides a service that allows developers to subsidize gas fees for their users.
 
-This concept, known as "Account Abstraction" in other chains, is a **built-in feature** in NEAR. User can wrap transactions in messages known as **meta-transaction**, that any other account can relay to the network.
+## Chain Abstraction: The Holistic View
 
-:::tip
-In NEAR the relayers simply attach NEAR to cover gas fees, and pass the transaction to the network. There, the transaction is executed as if the **user had sent it**.
-:::
+NEAR's ecosystem vision is to facilitate interactions across all blockchain networks and the larger internet. This transformative approach aims at enhancing the user experience and developer capabilities through advanced technologies, specifically in the realm of decentralized finance (DeFi) and artificial intelligence (AI).
```

Comment:
This sounds like another intro, maybe we don't need it, or we need to merge it above